Best practices/Vulnerable dependencies

```diff
@@ -0,0 +1,69 @@
+
+One important note about this command is that the packages are not scanned and analyzed, but rather checked against the centralized GitHub Advisory Database. This means that any non-public packages that might not be safe will not be marked as vulnerable. You can learn more about this in Microsoft's [blog post about the command](https://devblogs.microsoft.com/nuget/how-to-scan-nuget-packages-for-security-vulnerabilities/).
+
+## What to do about them?
+
+Now that we've ran the command, we have a list of packages with known vulnerabilities. The next step is determining whether that vulenrability is actually afflicting our system, and, if it is, how to fix it.
+
+### Analyzing the vulnerability
+
+This step of the process is unfortunately not straightforward. The result of the command gives us the links to the reports for each found vulnerability. The reports specify the severity and have some short descriptions of the exploit steps and what impact the vulnerability could have on a system.
+
+The severity can be minimal, medium, high, or critical. This level is determined by a couple of factors of potential vulnerabilities: how easy is it to set up, does it require some user interaction, how much data or control can a malicious attacker obtain, etc. This level should not be directly correlated with how urgently should the issue be fixed. For example, a medium level can mean that only one user's data can be accessed, and a critical one could leak all of our users' data. While one is obviously worse than the other, we shouldn't allow either of them.
+
+Another important thing to get from a security report is whether a certain vulnerability can be used against our system. If a vulnerability is afflicting a feature that is not being used by our solution, we don't have to be urgent when it comes to resolving the issue (e.g. a critical vulnerability found in `System.Common.Drawing` transitively used by an API which doesn't have any graphical features). There are many ways to find this out. Some packages are open-source, which enables us to analyze it and see how that package is using the vulnerable dependency, and then conclude whether that is applicable to our system. If the vulnerability afflicts the part of our solution which is not being deployed (e.g. our test tools and projects), we can safely assume that the public part of our system is not affected. But there will be situations where the available information will not be enough to come to any conclusions more meaningful than an educated guess, in which case it we should fix the issue either by updating the packages.
```

Comment:
Last sentence of this section `... in which case it we should fix the issue..`, I think that the `it` word should be removed, so we should just say `... in which case we should fix the issue...`. 

Also, at the end of that sentence, it says `... we should fix the issue either by updating the packages.` Why are we using the word `either` if we are only proposing one thing (updating the packages)?

Reply:
Another classic result of iterating through a single sentence 5 times, and then leaving it half-baked without even realizing :)